Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN INSTALL_PKGS=" \
       gcc git rpm-build make desktop-file-utils patch dnf-plugins-core \
       " && \
     dnf install --setopt=tsflags=nodocs -y $INSTALL_PKGS && \
-    cd assets/tuned/daemon && \
```

Comment:
Do we need this change?

Reply:
Yes. the updated dir is `assets/tuned/tuned`. look the the comment above

Reply:
OK, I don't have a big problem with this change, even though I don't think it is needed. What needs to work though is cloning/updating the submodule using the `update-tuned-submodule` target and that doesn't seem to work.